extensions/amp-list/0.1/amp-list.css

```diff
@@ -22,27 +22,31 @@ amp-list[load-more] [load-more-end].amp-visible {
   width: 100%;
 }
 
-[load-more-button], [load-more-failed], [load-more-loading] {
-  text-align: center;
+amp-list[load-more] [load-more-loading].i-amphtml-default-ui,
```

Comment:
Change `i-amphtml-default-ui` to `i-amphtml-list-load-more` since this is default ui for `amp-list` with `load-more` but not for all of `amphtml`.